src/GooStew.sol

```diff
@@ -45,12 +44,12 @@ contract GooStew is ERC20, BoringBatchable, Constants {
     uint32 internal _sumMultiples; // sum of emissionMultiples of all deposited gobblers
     mapping(address => GobblerDepositInfo) public gobblerDeposits;
 
-    constructor(address gobblers, address goo, address initialFeeRecipient)
+    constructor(IGobblers gobblers, IERC20 goo, address initialFeeRecipient)
         ERC20("Inflation-bearing Goo", "ibGOO", 18)
     {
         _gobblers = gobblers;
         _goo = goo;
-        IERC20(goo).approve(gobblers, type(uint256).max);
+        // goo.approve(gobblers, type(uint256).max);
```

Comment:
can you replace this line with a comment saying "ArtGobblers.addGoo(uint256) does not require approvals"